light_prop/lightfield.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+
+
+class LightField:
+    def __init__(self, amp, phase):
+        self.amplitude = amp
+        self.phase = phase
```

Comment:
I assume you discussed it and agreed on `A exp(i phi)` instead of `re` `im`? Why?